packages/babel-helper-function-name/src/index.js

```diff
@@ -2,7 +2,8 @@ import getFunctionArity from "babel-helper-get-function-arity";
 import template from "babel-template";
 import * as t from "babel-types";
 
-const buildPropertyMethodAssignmentWrapper = template(`
+const buildPropertyMethodAssignmentWrapper = template(
+  `
```

Comment:
@vjeux Would there be any interest in handling this like the array in https://prettier.github.io/prettier/#%7B%22content%22%3A%22var%20foo%20%3D%20bar(%5B%5Cn%20%20%7B%5Cn%20%20%20%20foo%3A%204%5Cn%20%20%7D%5Cn%5D)%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22doc%22%3Afalse%7D%7D

The template literal approach is certainly an option if not.